contrib/assets_tutorial/assets_tutorial.py

```diff
@@ -91,20 +90,19 @@ def sync_all(e1, e2):
 # Alternatively, you can set validatepegin=0 in their configs and not
 # run the bitcoin node, but it is necessary for fully validating the two way peg.
 
-# At beginning "immature balance" holds all funds until genesis is mature
-# Regtest chain starts with 21M elements coins as OP_TRUE which the wallet
-# understands. This is useful for testing basic functionality.
-# In Elements there is no block subsidy. All 21M output value in a mainnet
-# pegged sidechain are locked for pegging.
+# Regtest chain starts with 21M bitcoins as OP_TRUE which the wallet
+# understands. This is useful for testing basic functionality and for
+# blockchains that have no pegging functionality. "Policy" coins are required
+# for anti-DoS purposes as well as asset issuance, which consumes inputs for entropy.
```

Comment:
This sentence needs to be expanded to say what you mean by "policy" coins. At first glance it sounds like it's referring to standardness or something because the paragraph is about output scripts, but then I remembered policyAsset from the source code, which I guess is what you actually mean.

Maybe just replace `"Policy"` with `Initially existing` and the paragraph still works.

Reply:
No, I specifically meant coins which relay policy use for anti-DoS. I may want to rename the utility functions, since there are other reasons to seed the utxo set.